.github/ISSUE_TEMPLATE/5_feature_request.yml

```diff
@@ -0,0 +1,22 @@
+name: "Request a new Feature"
+description: "Create a request for a new feature"
+labels: ["meta:feature-request"]
+body:
+- type: textarea
+  attributes:
+      label: "Expected behavior of the wanted feature"
+      description: >
+          Before requesting a new feature make sure it hasn't been
+          [requested yet](https://github.com/mpv-player/mpv/labels/meta%3Afeature-request).
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Alternative behavior of the wanted feature"
+- type: textarea
+  attributes:
+      label: "Log File"
```

Comment:
Do we need log file in feature request?

I would prefer to ask about samples. Sometimes users ask for support of X format or something.

Reply:
i added it [specifically](https://github.com/mpv-player/mpv/commit/ea13ec6d985112dfb9611cd49bced369185a6494) at some point, because of a discussion we had on irc (i believe) because of platform dependent feature request and the log provided informations that were/would be helpful.

though i din't really have hard feeling about it.